express/styles/styles.css

```diff
@@ -516,6 +516,14 @@ main .section:last-of-type>.default-content-wrapper {
   padding-bottom: 40px;
 }
 
+main .section:first-of-type {
+  margin-bottom: -75px;
+}
```

Comment:
This feels very risky to me. Can we check again to make sure to target only the desired use case?